server/plugin/utils.go

```diff
@@ -312,6 +312,16 @@ func (p *Plugin) GetSubscriptionsForAccessibleChannelsOrProjects(subscriptionLis
 		return nil, err
 	}
 
+	if createdBy == constants.FilterCreatedByMe {
+		var filteredSubscriptionList1 []*serializers.SubscriptionDetails
```

Comment:
```suggestion
		var filteredSubscriptionList []*serializers.SubscriptionDetails
```

Reply:
and if there is issue with re-declaration you can use `filteredSubscriptionListForCurrentUser`